webpack/ForemanTasks/Components/TaskDetails/Components/TaskInfo.js

```diff
@@ -0,0 +1,234 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col, ProgressBar } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import EllipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import { timeInWords } from './TaskHelper';
+
+class TaskInfo extends Component {
+  isDelayed = () => {
+    const startAt = new Date(this.props.startAt);
+    const startedAt = new Date(this.props.startedAt);
+    startAt.setMilliseconds(0);
+    startedAt.setMilliseconds(0);
+    return startAt.getTime() !== startedAt.getTime();
+  };
+  resultIcon = () => {
+    if (this.props.state !== 'stopped') return 'task-status pficon-help';
+    const { result } = this.props;
+    let icon;
+    switch (result) {
+      case 'success':
+        icon = 'pficon-ok';
+        break;
+      case 'error':
+        icon = 'pficon-error-circle-o';
+        break;
+      case 'warning':
+        icon = 'pficon-ok status-warn';
+        break;
+      default:
+        icon = 'pficon-help';
+        break;
+    }
+    return `task-status ${icon}`;
+  };
+
+  progressBarType = () => {
+    const { result } = this.props;
+    switch (result) {
+      case 'error':
+        return 'danger';
+      case 'pending':
+        return null;
+      default:
+        return result;
+    }
+  };
+
+  render() {
+    const {
+      action,
+      endedAt,
+      result,
+      startAt,
+      startBefore,
+      startedAt,
+      state,
+      help,
+      output,
+      error,
+      progress,
+      username,
+      usernamePath,
+    } = this.props;
+
+    const details = [
+      [
+        {
+          title: 'Name',
+          value: <EllipsisWithTooltip>${action}</EllipsisWithTooltip>,
```

Comment:
This is not string interpolation, the dollar sign appears in the final text.

Reply:
Fixed Thanks!